src/compiler/build/watch-build.ts

```diff
@@ -53,6 +53,13 @@ export const createWatchBuild = async (
     buildCtx.hasHtmlChanges = hasHtmlChanges(config, buildCtx);
     buildCtx.hasServiceWorkerChanges = hasServiceWorkerChanges(config, buildCtx);
 
+    if (buildCtx.isRebuild) {
```

Comment:
Can we add a logging statement that occurs before we enter this `if` statement (so it always executes?) - I think it could be useful to see cases where `onBuild` is triggered with the `if` statement's conditional evaluating to `true` (which we'd see by the presence of the `console.log`s) and when it evaluating to `false` (which we'd see the proposed console log, but not the ones in the `if` statement)

Reply:
e.g. - https://github.com/ionic-team/stencil/compare/rwaskiewicz/ts-reload-bug-debug#diff-b3739ad1981b6aaadc57ffe972513cf24b1cda208aceccd8a642b0539fc8e878R88